sdk/storage/azblob/bloberror/error_codes.go

```diff
@@ -149,3 +149,8 @@ const (
 	UnsupportedQueryParameter                         Code = "UnsupportedQueryParameter"
 	UnsupportedXMLNode                                Code = "UnsupportedXmlNode"
 )
+
+// MissingSharedKeyCredential - Error is returned when SAS URL is being created without SharedKeyCredential.
```

Comment:
I believe the doc comment needs to be directly above the type declaration in order for the tooling to pick it up.

Reply:
Fixed!